inst/asa.csl

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Would it be okay to move files asa.csl and bibliography.bib to the vignettes folder?

Reply:
Or, does having them in `inst/` help with the citations in Rd files too?

Reply:
If we had references only in vignettes, I would prefer the `vignettes/` folder. However, since we use references elsewhere in the package documenation too, I suggest to use `inst/` as a central location. This is also how it was done in the `mmrm`package, see [here](https://github.com/openpharma/mmrm/tree/main/inst).